src/ucp/tag/rndv.c

```diff
@@ -846,6 +846,12 @@ static void ucp_rndv_req_send_rma_get(ucp_request_t *rndv_req, ucp_request_t *rr
 
     ucp_trace_req(rndv_req, "start rma_get rreq %p", rreq);
 
+    ucs_assertv(rreq->recv.length == rndv_rts_hdr->size,
```

Comment:
`rreq->recv.length >= rndv_rts_hdr->size`?

Reply:
this is for iodemo debugging where we need exact match

Reply:
I see

Reply:
but this is general code, maybe pass request flag that we expect exact match? what if user app does not have exact match?

Reply:
we don't have recv_nbx here.. maybe env var? UCX_TAG_MATCH_LENGTH_CHECK?